java/src/jmri/jmrit/beantable/BlockTableAction.java

```diff
@@ -582,15 +582,15 @@ protected void setDefaultSpeeds(JFrame _who) {
         blockSpeedCombo.setEditable(true);
 
         JPanel block = new JPanel();
-        block.add(new JLabel(Bundle.getMessage("BlockSpeedLabel")));
+        block.add(new JLabel(Bundle.getMessage("BlockSpeedLabel", ":")));
```

Comment:
This pattern is wrong. The formatter property string needs to be a grammatically complete statement, or a string that is complete within the dictates of the UI component (for example, a label that includes a keyboard nemonic as underlined in a Germanic or Romance language, would need to include that same nemonic in parenthesis in Japanese).